larpix/larpix.py

```diff
@@ -1110,7 +1111,7 @@ def read(self):
         else:
             warnings.warn('no IO object exists, no packets will be received', RuntimeWarning)
         if self.logger:
-            self.logger.record(packets, data_type='READ', timestamp=timestamp)
+            self.logger.record(packets, direction=1)
```

Comment:
I feel like this should maybe inherit from the hdf5 format... or maybe this should be specified in the larpix core code and then inherited by the hdf5 format

Reply:
Which part specifically? This was just an adaptation of your ``Logger`` implementation and I think you did the right thing with it so I'm reluctant to change it.

Reply:
Just the direction value, in case we need to change it in the future. Maybe not worth the effort.

Reply:
Per our discussion - I will switch from ``direction`` being 0 or 1 (hardcoded) to using ``Logger.READ`` and ``Logger.WRITE`` (which store the 0 and 1 in ``logger/__init__.py``)